### PR TITLE
Feature/acl

### PR DESCRIPTION
**Pull request by saqibraza**
l4 provider-type with working beaker tests

### CODE REVIEW

**Comment by bansalpradeep** [comment]
test comment

File: examples/demo_acl.pp [hunk 1]
```diff
@@ -16,29 +16,48 @@
...
   }
   cisco_ace { 'ipv4 my_ipv4_acl 10':
-    ensure                 => 'present',
-    action                 => 'permit',
-    proto                  => 'tcp',
-    src_addr               => '1.2.3.4 2.3.4.5',
-    src_port               => 'eq 40',
-    dst_addr               => '8.9.0.4/32',
-    dst_port               => 'range 32 56',
+    ensur    => 'present',
```
Comment (chrisvanheuveln) [inline examples/demo_acl.pp]: Have you tested these changes? I don't expect this to work with the typo above.

Comment (saqibraza) [inline examples/demo_acl.pp]: I kept playing around with the demo_acl on the master for testing. Sorry about this typo. I'll fix and test again.

Comment (chrisvanheuveln) [inline examples/demo_acl.pp]: Everything else looks good so just make that change and update your branch to latest and I'll merge it.

Comment (chrisvanheuveln) [inline examples/demo_acl.pp]: nm. I know you're leaving this project soon so I'll just merge it as is and fix it post-merge.

**Comment by chrisvanheuveln** [comment]
Closing this PR since I manually merged this code for Saqib.
Ref: https://github.com/cisco/cisco-network-puppet-module/commit/b869df59ecf15bd0b4d8fb66b1aa8e1cb1f37b5b

**Pull request closed without merging**